src/runtime/loader.ts

```diff
@@ -112,9 +114,13 @@ export async function loadStack(functionsDir: string): Promise<ManifestStack> {
 
   extractStack(mod, endpoints, requiredAPIs);
 
-  return {
+  var stack: ManifestStack = {
```

Comment:
```suggestion
  const stack: ManifestStack = {
```

Reply:
`var` is outdated. Always use `let` or `const`

Reply:
Done.